canvas_site_creator/templates/canvas_site_creator/create_new_course.html

```diff
@@ -0,0 +1,277 @@
+{% extends 'canvas_site_creator/base.html' %}
+
+{% load static %}
+{% load collections %}
+
+{% block css %}
+  <link rel="stylesheet" type="text/css" href="{% static 'datatables-bootstrap3/BS3/assets/css/datatables.css' %}" media="screen"/>
+  {% if debug %}
+    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.css' %}" media="screen"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'datatables-1.10.7/media/css/jquery.dataTables.css' %}" media="screen"/>
+  {% else %}
+    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" media="screen"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'datatables-1.10.7/media/css/jquery.dataTables.min.css' %}" media="screen"/>
+  {% endif %}
+{% endblock css %}
+
+{% block js %}
+  {% if debug %}
```

Comment:
yes, thank u! Correctly specified in restricted_access.html, but missed it here..